File: 07_Tuesday/prac.py
def prac():
    number_of_0 = 0
    number_of_1 = 0

    numbers = [0, 0, 1,1,1,1]
    for n in numbers:
        if n == 0:
            number_of_0 += 1
        elif (n == 1):
            number_of_1 += 1
    return number_of_1 > number_of_0


def more_prac():
    number_of_0 = 0
    number_of_1 = 0

    numbers = [0, 0, 1,1,1,1]

    n = 1
    while (n < len(numbers)):
        if n == 0:
            number_of_0 += 0
        elif (n ==1):
            number_of_1 += 1
        n += 1
    return number_of_1 > number_of_0



def main():
    print(prac())
    print(more_prac())


if __name__ == "__main__":
    main()

File: 07_Tuesday/prac.js
function prac() {
    numberOf0 = 0
    numberOf1 = 0

    numbers = [0, 0, 1, 1, 1, 1]

    for (n of numbers) {
        if (n == 0) {
            numberOf0 += 1;
        } else if (n == 1) {
            numberOf1 += 1;
        }
    }
    return numberOf1 > numberOf0
}

function morePrac() {
    numberOf0 = 0
    numberOf1 = 0

    numbers = [0, 0, 1, 1, 1, 1]

    numbers.forEach(element => {
        if (element == 0) {
            numberOf0 += 1;
        } else if (element == 1) {
            numberOf1 += 1;
        }
    });
    return numberOf1 > numberOf0
}

console.log(prac())
console.log(morePrac())